src/main/java/teammates/ui/template/InstructorFeedbackResultsSectionPanel.java

```diff
@@ -0,0 +1,146 @@
+package teammates.ui.template;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class InstructorFeedbackResultsSectionPanel {
+    private String panelClass;
+    private boolean isGroupedByTeam;
+    private boolean isDisplayingMissingParticipants;
+    private boolean isLoadSectionResponsesByAjax;
+    
+    private String sectionName;
+    private String sectionNameForDisplay;
+    
+    private String statisticsHeaderText;
+    private String detailedResponsesHeaderText;
+    
+    // A mapping from team name to a list of participant panels. Each participant panel is for one member of the team
+    private Map<String, List<InstructorResultsParticipantPanel>> participantPanels;
+    
+    private boolean isDisplayingTeamStatistics;
+    private Map<String, List<InstructorResultsQuestionTable>> teamStatisticsTable;
+    private Map<String, Boolean> isTeamWithResponses;
+
+    public InstructorFeedbackResultsSectionPanel() {
+        panelClass = "panel-success";
```

Comment:
Should we be storing this here?
I'm not completely sure, but it seems to never change (searched for references to the variable / the setter), so just putting it into the tag might be just as valid.